xml/art_migration.xml

```diff
@@ -395,8 +396,20 @@ done</screen>
     <itemizedlist>
      <listitem>
       <para>
-       For non-DVR clouds, evacuate any existing &o_netw; L3 routers on the
-       seed node:
+       In Distributed Virtual Router (DVR) enabled clouds, responsibility for
+       network routing is shared across the compute nodes, and downtime of the
+       seed node as a controller will have no adverse effect on workload network
+       traffic. To see if DVR is enabled in your cloud, run the following command
+       and check the entries in the "Distributed" column.
+      </para>
+<screen><prompt>(old-deployer)> </prompt>source ~/service.osrc
+<prompt>(old-deployer)> </prompt>neutron router-list</screen>
+      <para>
+       If you observe non-distributed routers, then the next step will ensure
```

Comment:
I don't have access to a CVR cloud to check, but is it possible to qualify the statement "If you observe non-distributed routers" more, for example to "If you observe non-distributed routers running on the seed node"? If so we should do that. Looks good otherwise.

Reply:
To me, it looks like the output of "neutron router-list" does not indicate to the user which controller node a CVR router is running on. So, it's safest to run the evacuation script any time you see non-distributed routers (which should be a no-op if the planned seed node is not hosting any CVR routers anyway).